release/ray_release/scripts/build_pipeline.py

```diff
@@ -185,6 +186,9 @@ def main(test_collection_file: Optional[str] = None, no_clone_repo: bool = False
             else:
                 this_ray_wheels_url = ray_wheels_url
 
+            ray_commit = parse_commit_from_wheel_url(this_ray_wheels_url)
+            if ray_commit:
+                env.update({"RAY_TEST_COMMIT": ray_commit})
```

Comment:
can we parse and set RAY_TEST_BRANCH as well

Reply:
Hmmm, I don't think that's intended for RAY_TEST_BRANCH.
Consider the scenario that you want the wheel to be whatever is newest on ray master branch but you are actually testing some updated code in `tune_cloud_test.py`. 
In this case, `RAY_TEST_REPO` is your forked repo of ray. `RAY_TEST_BRANCH` is the branch you are working on for updating `tune_cloud_test.py`.

Reply:
For this exact reason, maybe I shouldn't name it RAY_TEST_COMMIT. It's misleading. 

This var is intended to be just parsed the commit from wheel url.

Whether to use it as test commit is up to other gating factors.